src/utils/oauth/callBack.ts

```
@@ -0,0 +1,43 @@
+  const router = useRouter();
+  const params = useSearchParams();
+  const code = params.get('code');
+
+  useEffect(() => {
+    if (!code) {
+      throw new Error('인가 코드가 없습니다.');
+      return;
+    }
+
+    const body: OAuthRequest = {
+      redirectUri: `${process.env.NEXT_PUBLIC_APP_URL}/auth/callback/kakao`,
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)